contentcuration/contentcuration/static/less/content-container.less

```diff
@@ -548,6 +548,9 @@ body {
                   &.kind-html5 {
                     background-color: @html-color;
                   }
+                  &.kind-slideshow {
```

Comment:
You'll also need to add the styling here: https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/static/less/content-container.less#L652 (can see when you switch the view mode to compact in the blue toolbar)